server/cmd/bblfsh-web/main.go

```diff
@@ -67,13 +67,13 @@ func main() {
 
 	for _, a := range assets {
 		if a != "static" {
-			r.StaticFile("/"+a, filepath.Join(dir, a))
+			r.StaticFile(a, filepath.Join(dir, a))
 		}
 	}
 	indexPath := filepath.Join(dir, "index.html")
-	r.StaticFile("/", indexPath)
-	r.Static("/static", filepath.Join(dir, "static"))
-	server.Mount(s, r.Group("/api"))
+	r.StaticFile("", indexPath)
```

Comment:
do we really need to remove slashes here? It's a server that always gets requests on `/`, no?

Reply:
you're right @smacker! thanks!